config_test.go

```diff
@@ -22,7 +22,7 @@ import (
 
 func TestHideConfigSecrets(t *testing.T) {
 	sc := &SafeConfig{}
-	err := sc.ReloadConfig([]string{"testdata/snmp-auth.yml"}, false)
+	err := sc.ReloadConfig(nil, []string{"testdata/snmp-auth.yml"}, false)
```

Comment:
Use `promslog.NewNopLogger()` from `github.com/prometheus/common/promslog` instead of nil here.